model/OrganisationCreateRequest.ts

```diff
@@ -3,3 +3,9 @@ import { OrganisationUpdateRequest } from "./OrganisationUpdateRequest"
 export interface OrganisationCreateRequest extends OrganisationUpdateRequest {
 	code: string
 }
+
+export namespace OrganisationCreateRequest {
+	export function is(value: OrganisationCreateRequest | any): value is OrganisationCreateRequest {
+		return typeof value == "object" && typeof value.code == "string" && OrganisationUpdateRequest.is(value)
```

Comment:
before i had a problem regarding the type system when using an is-function directly on value (maybe it works here), but we can do .is({...value}) to be safe